lib/auth/integration/integrationv1/awsoidc.go

```diff
@@ -495,6 +495,58 @@ func (s *AWSOIDCService) DeployDatabaseService(ctx context.Context, req *integra
 	}, nil
 }
 
+// ListDeployedDatabaseServices deploys Database Services into Amazon ECS.
+func (s *AWSOIDCService) ListDeployedDatabaseServices(ctx context.Context, req *integrationpb.ListDeployedDatabaseServicesRequest) (*integrationpb.ListDeployedDatabaseServicesResponse, error) {
+	authCtx, err := s.authorizer.Authorize(ctx)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	if err := authCtx.CheckAccessToKind(types.KindIntegration, types.VerbUse); err != nil {
```

Comment:
should we have an authz check on kind db_service or anything?

Reply:
No strong opinion on that.
We don't require any other authz check for deploying.
We have been requiring only `integration.use` check for those methods, but we can easily add more.
If we add it here, then I think we should add it to the deployment call.

The most sensitive part of this listing is the `teleport.yaml` configuration, but even that only has the IAM Token Name, which we don't consider to be a secret.

I'm in favor of keeping it as is, but let me know what you think.